lib/msal-browser/src/config/Configuration.ts

```diff
@@ -278,6 +282,7 @@ export function buildConfiguration(
         },
         skipAuthorityMetadataCache: false,
         supportsNestedAppAuth: false,
+        extraTokenRequestHeaders: {},
```

Comment:
And if we need to support this, I will say adding it as an override in network implementation would be a better place, however that places a dependency on the client choosing a network implementation which isn't true always. Hence it may make it a little more complex implementation.